README.md

```diff
@@ -55,6 +62,13 @@ app.use(session({
 }));
 ```
 
+## Expiration
+If this library `get()`s an expired session this library will delete it from
```

Comment:
I think we should explain the feature here:

"""
If a session is fetched with the delta between the `createdAt` time and current time greater than `expirationMs`,
the session will not be returned and will instead be destroyed.
"""

with the caveat below that:

```
Datastore does not support a `ttl`, and tokens are only deleted if a session is fetched. You will likely want to implement logic to occasionally delete expired sessions.
```

Reply:
okay, updating to:

```markdown
## Expiration
If a session is fetched with the delta between the createdAt time and current
time greater than expirationMs, the session will not be returned and will
instead be destroyed.

Datastore does not support a `ttl`, and tokens are only deleted if a session
is fetched. You will likely want to implement logic to occasionally delete
expired sessions.
```

Reply:
This has been updated in the README, please confirm this is what you wanted.